api/maven-api-core/src/main/java/org/apache/maven/api/DependencyScope.java

```diff
@@ -28,9 +28,9 @@
 import org.apache.maven.api.annotations.Nonnull;
 
 /**
- * Dependency scope.
- * This represents at which time the dependency will be used, for example, at compile time only,
- * at run time or at test time.  For a given dependency, the scope is directly derived from the
+ * Represents at which time the dependency will be used.
+ * If may be, for example, at compile time only, at run time or at test time.
```

Comment:
`Indicates when the dependency will be used. For example, it may be at compile time only, at runtime, or at test time.`